docs/content/Upgrade/upgrade4.md

```diff
@@ -0,0 +1,273 @@
+
+A major change to this container is that the PostgreSQL process is now managed by Patroni. This allows a PostgreSQL cluster that is deployed by the PostgreSQL Operator to manage its own uptime and availability, to elect a new leader in the event of a downtime scenario, and to automatically heal after a failover event.
+
+When creating your new clusters using version 4.3.0 of the PostgreSQL Operator, the `pgo create cluster` command will automatically use the new `crunchy-postgres-ha` image if the image is unspecified. If you are creating a PostGIS enabled cluster, please be sure to use the updated image name, as with the command:
+```
+pgo create cluster mygiscluster --ccp-image=crunchy-postgres-gis-ha
+```
+
+##### NOTE: As with any upgrade procedure, it is strongly recommended that a full logical backup is taken before any upgrade procedure is started. Please see the [Logical Backups](/pgo-client/common-tasks#logical-backups-pg_dump--pg_dumpall) section of the Common Tasks page for more information.
+
+The Ansible installation upgrade procedure is below. Please click [here](/upgrade/upgrade4#bash-installation-upgrade-procedure) for the Bash installation upgrade procedure.
+
+### Ansible Installation Upgrade Procedure
+
+Below are the procedures for upgrading the PostgreSQL Operator and PostgreSQL clusters using the Ansible installation method.
+
+##### Prerequisites.
+You will need the following items to complete the upgrade:
+
+* The latest 4.3.0 code for the Postgres Operator available
+
+These instructions assume you are executing in a terminal window and that your user has admin privileges in your Kubernetes or Openshift environment.
+
+##### Step 0
+For the cluster(s) you wish to upgrade, record the cluster details provided by
+
+        pgo show cluster <clustername>
+
+so that your new clusters can be recreated with the proper settings.
+
+##### Step 1
+For the cluster(s) you wish to upgrade, scale down any replicas, if necessary (see `pgo scaledown --help` for more information on command usage) page for more information), then delete the cluster
+
+        pgo delete cluster <clustername>
+
+##### Please note the name of each cluster, the namespace used, and be sure not to delete the associated PVCs or CRDs!
+
+
+##### Step 2
+
+Save a copy of your current inventory file with a new name (such as `inventory.backup)` and checkout the latest 4.3.0 tag of the Postgres Operator.
+
+
+##### Step 3
+Update the new inventory file with the appropriate values for your new Operator installation, as described in the [Ansible Install Prerequisites]( {{< relref "installation/install-with-ansible/prerequisites.md" >}}) and the [Compatibility Requirements Guide]( {{< relref "configuration/compatibility.md" >}}).
```

Comment:
Please keep a newline between headers and text.

Reply:
updated throughout the documentation updates